vgo/src/main/kotlin/com/jzbrooks/vgo/vd/VectorDrawableCommandPrinter.kt

```diff
@@ -235,7 +236,7 @@ class VectorDrawableCommandPrinter(private val decimalDigits: Int) : CommandPrin
     private fun print(point: Point) =
         buildString {
             append(print(point.x))
-            append(',')
```

Comment:
I want to say there was a reason this was omitted only for vector drawables on first implementation. I vaguely remember there being some issues with framework implementations of vector drawable on android for SDK versions ~21 (avoided with compatibility lib).

I searched around the issue tracker and couldn't find evidence of this.

This prevents some vectors from being represented more verbosely than inputs, and it probably isn't an issue now in practice.